packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.stories.ts

```diff
@@ -113,11 +117,79 @@ export const PublicationCard: Story = {
   },
 }
 
+const genieSelectedFacetConfigs: SelectedFacetConfig[] = [
+  {
+    facetColumnName: 'cohort',
+    facetValueColumnName: 'Cohort',
+  },
+  {
+    facetColumnName: 'version',
+    facetValueColumnName: 'version',
+  },
+]
+
+export const ReleaseCardLarge: Story = {
+  args: {
+    // TODO - update to GENIE release table (syn53978783) once data is updated
+    sql: 'select * from syn53701326 where isCurrentRelease = true',
+    type: RELEASE_CARD,
+    limit: 3,
+    releaseCardConfig: {
+      cardSize: 'large',
+      prependRelease: false,
+      releaseMetadataConfig: {
+        releaseDateColumnName: 'ReleaseDate',
+        releaseEntityColumnName: 'id',
+        releaseNameColumnName: 'name',
+      },
+      statsConfig: [
+        { columnName: 'Patients', label: 'Patients' },
+        { columnName: 'Samples', label: 'Samples' },
+      ],
+      primaryBtnConfig: {
+        label: 'Explore Data Release',
+        sourcePathColumnName: 'releaseExplorePath',
+        sourceExploreDataSqlColumnName: 'exploreDataSql',
+        selectedFacetConfigs: genieSelectedFacetConfigs,
+      },
+      secondaryBtnConfig: {
+        label: 'View Data Guide',
+        sourcePathColumnName: 'releaseExplorePath',
+        sourceExploreDataSqlColumnName: 'exploreDataSql',
+        selectedFacetConfigs: genieSelectedFacetConfigs,
+        staticSelectedFacets: [
+          {
+            facet: 'dataType',
+            facetValue: 'data_guide',
+          },
+        ],
+      },
+    },
+  },
+}
```

Comment:
Demonstration of the GENIE release card buttons linking to filtered explore pages using an example source table. GENIE releases link to GENIE explore page. GENIE-BPC releases link to GENIE-BPC explore page. Both primary and secondary buttons filter to the selected cohort and version. The secondary button also filters to only data_guide files.

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/108b2c8e-0df5-47d3-94b2-5178b6e083d2